legend-engine-xts-memsqlFunction/legend-engine-xt-memsqlFunction-api/src/main/java/org/finos/legend/engine/language/memsql/deployment/MemSqlFunctionArtifactGenerationExtension.java

```diff
@@ -0,0 +1,46 @@
+// Copyright 2023 Goldman Sachs
```

Comment:
Any reason why we have this just empty? should we remove it

Reply:
It was a duplicate class hence removed.